lib/factory_trace/helpers/converter.rb

```diff
@@ -21,7 +21,8 @@ def factory(factory)
         FactoryTrace::Structures::Factory.new(
           factory.name.to_s,
           factory.send(:parent).respond_to?(:name) ? factory.send(:parent).name.to_s : nil,
-          factory.defined_traits.map(&:name).map(&:to_s)
+          factory.defined_traits.map(&:name).map(&:to_s),
+          factory.instance_eval('@aliases').map(&:to_s)
```

Comment:
We save `names` and rewrite other code to use `names`? 

It seems like public API of `FactoryBot::Factory`
```
def names
      [name] + @aliases
    end
```

Reply:
Oh, thanks. I didn't notice `names` let me update the code